src/app/mini-games/custom-mini-game/services/sound.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SoundService {
+  isSoundOn = false;
+
+  constructor() {}
+
+  toggleSound(): void {
+    this.isSoundOn = !this.isSoundOn;
+  }
+
+  playAudio(sound: string): void {
+    let audio = new Audio();
+    if (sound === 'move') {
```

Comment:
@suggestion
```switch/case``` construction would be more suitable here

Reply:
As an option I'd propose to create map with key sound and value path to an asset